codebundles/aws-c7n-compliance/sli.robot

```diff
@@ -0,0 +1,95 @@
+*** Settings ***
+Metadata            Author   saurabh3460
+Metadata            Supports    AWS    Tag    CloudCustodian
+Metadata            Display Name    AWS Compliance
+Documentation        Counts the number of AWS resources that do not follow compliance.
+Force Tags    Tag    AWS    Ec2     S3      RDS      VPC      iam-group        iam-policy     iam-role        iam-user
+
+Library    RW.Core
+Library    RW.CLI
+Library    CloudCustodian.Core
+
+Suite Setup    Suite Initialization
+
+
+*** Tasks ***
+Check for Missing Tags on AWS ${AWS_RESOURCE_PROVIDERS} resources in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}` 
```

Comment:
Task title is too long. 
I'd suggest 
`Check for Missing AWS Resource Tags in AWS Region `${AWS_REGION}` for AWS Account `${AWS_ACCOUNT_ID}``